leaf-ui/js/displayAnalysis.js

```diff
@@ -3,6 +3,15 @@
  * that the web application would receive from the back end.
```

Comment:
Probably want to update this docstring to include reference to the analysis config bar eventually

Reply:
Done, will add more detail in future iterations once code is more set